exercises/bank.js

```diff
@@ -1,25 +1,51 @@
 
 
 class Bank{
-	constructor(){
+	constructor(bank_name){
+		this.bank_name = bank_name;
+		this.all_accounts = {};
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
 	}
-	makeAccount( ){
+	makeAccount(accountNum ){
+		if(this.all_accounts[accountNum]){
```

Comment:
for more efficiency, using this.all_accounts.hasOwnProperty can save you from wasted prototypal chain searching for something that doesn't exist